test/test_save_load_model_to_file.py

```diff
@@ -0,0 +1,32 @@
+from numpy.testing import assert_array_equal
+
+from training.create_model import get_fitted_model
+from training.creating_dataset import create_dataframe
+from training.save_load_model_to_file import load_model_from_file, save_model_to_file
+
+
+def test_save_load_model_to_file_default():
+    X = create_dataframe()
+    X.pop("book")
+    y = X.pop("level")
```

Comment:
to tworzenie X i y powtarza ci się w obu funkcjach, może warto to wydzielić z użyciem `@pytest.fixture` do osobnej funkcji

Reply:
Dobra sugestia